hazelcast-jet-kafka/src/main/java/com/hazelcast/jet/connector/kafka/ReadKafkaP.java

```diff
@@ -145,33 +155,62 @@ public void close() {
      * @param deserializeKey         function for deserializing keys
      * @param deserializeValue       function for deserializing values
      */
-    static <K, V> ProcessorMetaSupplier readKafka(String zkAddress, String groupId, String topicId,
-                                                  String brokerConnectionString,
-                                                  Function<byte[], K> deserializeKey,
-                                                  Function<byte[], V> deserializeValue) {
+    public static <K, V> ProcessorMetaSupplier readKafka(String zkAddress, String groupId, String topicId,
+                                                         String brokerConnectionString,
+                                                         Function<byte[], K> deserializeKey,
+                                                         Function<byte[], V> deserializeValue) {
+        return readKafka(zkAddress, groupId, topicId, brokerConnectionString, deserializeKey, deserializeValue,
+                CURRENT_POSITION_OFFSET_MAPPER);
+    }
+
+    /**
+     * Returns a meta-supplier of processors that consume a kafka topic and emit
+     * items from it as {@code Map.Entry} instances.
+     *
+     * <p>
+     *     You can specify a partition to offset mapper to mark the start offset of each partition.
+     *     Any negative value as an offset will throw {@code IllegalArgumentException}
+     * </p>
```

Comment:
I'm missing documentation that explains the meaning of the left and right operand of `BiFunction`.